binsync/core/client.py

```diff
@@ -346,6 +346,12 @@ def get_state(self, user=None, version=None, priority=None, no_cache=False):
         except MetadataNotFoundError:
             if user == self.master_user:
                 state = State(self.master_user, client=self)
+        except Exception as e:
+            if user == self.master_user:
+                raise
```

Comment:
Is this legal? You should probably do `raise e`

Reply:
Yes, it is. Rethrowing the named exception will clobber the original backtrace. See e.g. https://stackoverflow.com/questions/36153805/difference-between-raise-and-raise-e for more details